kaldb/src/main/proto/kaldb_configs.proto

```diff
@@ -192,6 +192,7 @@ message ManagerConfig {
 // Config for the recovery node.
 message RecoveryConfig {
   ServerConfig server_config = 1;
+  KafkaConfig kafka_config = 10;
```

Comment:
I don't love that we allow for divergence of some Kafka settings (e.g., bootstrap servers) that should never diverge, but may not be worth fragmenting the config structure to disallow that.

Reply:
yeah! But having it top level means it's leaky and potentially the manager can try to access these settings and fail in weird ways?